### PR TITLE
Added Models and Services

### PR DESCRIPTION
**Pull request by rfeseha**
I wrote the code for both service and model. I don't know if I have missed something or need more code so you guys can review and see

### CODE REVIEW

File: WebMvc/Services/CatalogService.cs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+               }
+           };
+
+           var categories = JArray.Parse(dataString);
+            foreach (var category in categories)
+            {
+                items.Add(
+                    new SelectListItem
+                    {
+                        Value = categories.Value<string>("id"),
```
Comment (vlau26) [inline WebMvc/Services/CatalogService.cs]:
```suggestion
                        Value = category.Value<string>("id"),
```

File: WebMvc/Services/CatalogService.cs [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+                        Text = categories.Value<string>("category")
```
Comment (vlau26) [inline WebMvc/Services/CatalogService.cs]:
```suggestion
                        Text = category.Value<string>("category")
```

File: WebMvc/Services/CatalogService.cs [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+            var locations = JArray.Parse(dataString);
+            foreach (var location in locations)
+            {
+                items.Add
+                    
+                   (
+                    new SelectListItem
+                    {
+                        Value = locations.Value<string>("id"),
```
Comment (vlau26) [inline WebMvc/Services/CatalogService.cs]:
```suggestion
                        Value = location.Value<string>("id"),
```

File: WebMvc/Services/CatalogService.cs [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+                        Text = locations.Value<string>("location")
```
Comment (vlau26) [inline WebMvc/Services/CatalogService.cs]:
```suggestion
                        Text = location.Value<string>("location")
```

File: WebMvc/Services/ICatalogService.cs [hunk 5]
```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.AspNetCore.Mvc.Rendering;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using WebMvc.Models;
+
+namespace WebMvc.Services
+{
+    interface ICatalogService
```
Comment (vlau26) [inline WebMvc/Services/ICatalogService.cs]:
```suggestion
    public interface ICatalogService
```

**Comment by parks38** [comment]
If you made changes to vivian's comments I think the code should be fine!

**Comment by parks38** [comment]
> If your done with making changes to vivian's comments, it's good to go! Nice Work!

I'll leave it as approve so you can merge it if all the revisions are changed.

**Pull request merged**